stellarphot/core.py

```diff
@@ -1326,9 +1334,128 @@ def prepare_cat(cat):
         colname_map=vsx_map,
         prepare_catalog=prepare_cat,
         no_catalog_error=True,
+        tidy_catalog=False,
     )
 
 
+def refcat2(field_center, radius=1 * u.degree, clip_by_frame=False, padding=100):
+    """
+    Return the items from APASS DR9 that are within the search radius and
+    (optionally) within the field of view of a frame.
+
+    Parameters
+    ----------
+    field_center : `astropy.coordinates.SkyCoord`, `astropy.wcs.WCS`, or FITS header
+        Either a `~astropy.coordinates.SkyCoord` object, a `~astropy.wcs.WCS` object
+        or a FITS header with WCS information. The input coordinate should be the
+        center of the frame; if a header or WCS is the input then the center of the
+        frame will be determined from the WCS.
+
+    radius : `astropy.units.Quantity`, optional
+        Radius around which to search.
+
+    clip_by_frame : bool, optional
```

Comment:
Is this needed due to the catalog search being a circular region and the frame being square?

Reply:
Yeah; the search is always a cone, but it can be trimmed down to a rectangle if desired.